package.json

```diff
@@ -5,15 +5,41 @@
     "type": "git",
     "url": "https://github.com/brownhci/WebGazer.git"
   },
-  "devDependencies": {
-    "git": "^0.1.5",
-    "grunt": "*",
-    "grunt-contrib-concat": "*",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-uglify-es": "*",
-    "jsdoc": "^3.6.3"
+  "license": "LGPL-3.0-or-later",
+  "main": "dist2/webgazer.commonjs2.js",
+  "module": "src/index.mjs",
+  "scripts": {
+    "clean": "rimraf dist2",
+    "prebuild": "npm run clean",
+    "dev": "webpack --progress --colors --config webpack.config.js --mode=development",
+    "build": "webpack --progress --colors --config webpack.config.js --mode=production"
+  },
+  "files": [
+    "dist2",
+    "src"
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run build"
```

Comment:
```suggestion
      "pre-commit": "npm run dev"
```

Reply:
it is not necessary, generated output lives in `dist2/` and `dist/` directory is for users who want to test this repo's examples out without installing the dependencies. file size becomes more than 4MB when we run `npm run dev`(dev mode). when we run `npm run build(prod mode) file size become ~1.5MB(its because it is bundled with all the deps).

Reply:
Ah, sorry my mistake, did not notice dist2 and I thought the committed things were binary because of the source maps, (no reason for those to be committed for unminified JS, right? removing those should make it more or less all set.

Reply:
By having those source maps we get to see where is the error in actual source files otherwise we get to see the error line number in the bundled file. 

eg. I randomly put var m; in method begin, and then I get to see original source file(index.mjs) from the bundled file.
![Screenshot 2020-08-05 at 9 59 11 AM](https://user-images.githubusercontent.com/24789277/89371969-81742380-d702-11ea-8d4f-7030d48348e0.png)

Reply:
@fxnoob 
Yeah, I absolutely agree they are useful for development. I still want them generated, just not committed to the git repository. (They will be uploaded along with the minified version to the website for releases.

(if you are developing locally, you probably will run npm at least once to generate them before opening the console nad looking for error, no?)

Reply:
right! makes sense to me.